wallet/how-to/use-non-evm-networks/starknet/manage-starknet-accounts.md

```diff
@@ -0,0 +1,412 @@
+---
+description: Manage Starknet accounts in MetaMask.
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Manage Starknet accounts
+
+You can manage Starknet accounts in MetaMask using the
+[`get-starknet`](https://github.com/starknet-io/get-starknet) library or the
+[`wallet_invokeSnap`](/snaps/reference/wallet-api-for-snaps/#wallet_invokesnap) JSON-RPC method.
+
+:::note Notes
+
+- Account creation in Starknet is handled by the wallet provider.
+  As a dapp developer, you do not create accounts directly.
+  Instead, you can guide users to [create an account](connect-to-starknet.md) with MetaMask.
+- Currently, the Starknet Snap doesn't support multiple Starknet accounts.
+
+:::
+
+## Prerequisites
+
+[Connect to Starknet](connect-to-starknet.md) from your dapp.
+
+## Display account information
+
+After a user connects to their Starknet account in MetaMask, you can display the account details.
+The following example displays the account address: 
+
+<Tabs>
+  <TabItem value="get-starknet" default>
+
+  ```javascript
+  const showAccountInfo = async () => {
+    const account = await connectStarknetAccount();
+    if (account) {
+      document.getElementById("accountAddress").innerText = `Account Address: ${account}`;
+    }
+  };
+  ```
+
+  </TabItem>
+  <TabItem value="wallet_invokeSnap">
+  
+  ```javascript
+  const showAccountInfo = async () => {
+    if (typeof window.ethereum !== "undefined" && window.ethereum.isMetaMask) {
+      try {
+        // Invoke the Starknet Snap to get account information.
+        const response = await window.ethereum.request({
+          method: "wallet_invokeSnap",
+          params: {
+            snapId: "npm:@starknet-snap/snap", 
+            request: {
+              method: "starknet_getAccounts"
```

Comment:
This method `starknet_getAccounts` is used in several examples but is not documented in the reference. Is there a spec for it?